### PR TITLE
Desktop access: add option to send ctrl+alt+del

### LINKED ISSUE #41448: Desktop Access: Add menu option to send ctrl+alt+del to remote server
Many remote desktop and virtualization tools include an option to send a ctrl+alt+del sequence to the remote server because a Windows host will handle this key sequence before a remote desktop client gets the chance to. We should to the same.

This would solve a subset of #40205.

### PR DESCRIPTION
**Pull request by zmb3**
Closes #41448

Changelog: Add an option to send a Ctrl+Alt+Del sequence to remote desktops.

### CODE REVIEW

**Comment by zmb3** [comment]
Screenshot 2024-05-12 at 10 47 16 AM

File: web/packages/teleport/src/DesktopSession/useDesktopSession.tsx [hunk 1]
```diff
@@ -185,6 +186,15 @@ export default function useDesktopSession() {
     }
   };
 
+  const onCtrlAltDel = () => {
+    if (!tdpClient) {
+      return;
+    }
+    tdpClient.sendKeyboardInput('ControlLeft', ButtonState.DOWN);
+    tdpClient.sendKeyboardInput('AltLeft', ButtonState.DOWN);
+    tdpClient.sendKeyboardInput('Delete', ButtonState.DOWN);
```
Comment (ibeckermayer) [inline web/packages/teleport/src/DesktopSession/useDesktopSession.tsx:195]: Sort of worried that not sending an "UP" could cause weird issues, but I haven't been able to induce anything.

**Review by ibeckermayer — COMMENTED** [review]
The icon looks like a black square to me in light mode. It's also a little "bold" compared to the other icons in dark mode, maybe there's a way to invert the light/dark that could make it appear more inline with the others.
image

File: web/packages/teleport/src/DesktopSession/useDesktopSession.tsx (hunk 1 above)
Comment (zmb3) [reply web/packages/teleport/src/DesktopSession/useDesktopSession.tsx:195]: Agreed, I thought I was going to have to do that too, but this is the first thing I tried and I couldn't get it to break!

**Comment by zmb3** [comment]
@ibeckermayer the icon has been fixed, good catch

**Review by ibeckermayer — APPROVED** [review]

**Review by rudream — APPROVED** [review]

**Review by ravicious — APPROVED** [review]

**Pull request merged**